manifests/type/capactio/capact/validation/upload.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         "title": "The schema for upload Type",
         "examples": [
```

Comment:
The Implementation `passing-a` uploads TypeInstance with `key: testValue`. Should the `passing-b` Implementation be changed as well? Cause I see there:

```yaml
              args: ["echo 'Implementation B' && cp /input /upload && echo 'key: true' > /expected && diff /expected /injected && sleep 1"]

```

Reply:
Not really. In our e2e tests, we examine if the expected is the same as injected for the value `key: true` so it is ok.

Reply:
Oh, right 👍  I read the code in wrong way. Sounds good 👍